tests/ios/frameworks/with-bundle-id/BUILD.bazel

```diff
@@ -0,0 +1,12 @@
+load("//rules:framework.bzl", "apple_framework")
+
+apple_framework(
```

Comment:
Actually to properly test this we need to set resource bundles here, the bundle identifier was also missing in the fmw regarding but I'll fix that so this test is testing the right thing (then update the PR description/screenshot)

Reply:
Done